app/models/node.rb

```diff
@@ -71,7 +71,7 @@ def updated_month
     updated_at.strftime('%B %Y')
   end
 
-  has_many :revision, foreign_key: 'nid' # , dependent: :destroy # re-enable in Rails 5
+  has_many :revision, foreign_key: 'nid', dependent: :destroy
```

Comment:
@jywarren but here :arrow_up: I have added this `dependent: :destroy`. Should I remove this?

Reply:
uh oh - oops, i missed this! Um, I think this is OK. I read the docs carefully. Thanks!

Reply:
I also peeked :mag: at this :arrow_right: https://guides.rubyonrails.org/association_basics.html while changing so this one should be fine.